### PR TITLE
Release CI

### PR DESCRIPTION
**Pull request by jswift-stfc**
Added release pipeline.
Added version to gudpy.spec.

### CODE REVIEW

**Comment by jswift-stfc** [comment]
Expect PR CI to fail, since I deleted ReleaseNotes.md; since it is specific to `release/*` branches, and here we want to merge into the`master/main` branch.

File: .azure-pipelines/release.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
+trigger:
+  branches:
+    include:
+      - release/*
+
+pr: none
+
+variables:
+  tag: 2021.1
```
Comment (trisyoungs) [inline .azure-pipelines/release.yml:9]: For clarity, renaming `tag` to `gudrunTag` would make this more transparent (certainly for me!).

**Review by trisyoungs — APPROVED** [review]
All good bar one small suggestion.

File: .azure-pipelines/release.yml (hunk 1 above)
Comment (jswift-stfc) [reply .azure-pipelines/release.yml:9]: Sounds good! I'll update this across all pipelines.

**Pull request merged**